pkg/config/config.go

```diff
@@ -74,8 +74,8 @@ var (
 			Required:   true,
 		},
 		{
-			Name:     "exclude_nodes_by_label",
-			Help:     "exclude jenkins nodes by label",
+			Name:     "find_by_node_label_name",
```

Comment:
Suggest this name for the param. By the way the right terminalogy for `jenkins` is `agent` and not `nodes`.

```suggestion
			Name:     "agent_with_label",
```

Reply:
<img width="872" alt="Screenshot 2023-01-12 at 9 50 07" src="https://user-images.githubusercontent.com/1666303/212008558-dfc286f0-e8bb-465d-b54c-218399445b3b.png">
 i"m not sure, there is know meaning of agents in jenkins only nodes,